hibernate-core/src/main/java/org/hibernate/tool/schema/internal/DdlTransactionIsolatorProvidedConnectionImpl.java

```diff
@@ -0,0 +1,47 @@
+package org.hibernate.tool.schema.internal;
```

Comment:
Oops.  I was using a "parallel" clone of master for this work (so as to not disrupt my 6.0 work) which did not have the Copyright plugin set up.  Surprised I only missed one tbh :)  Good catch!